purchase_order_product_recommendation_secondary_unit/wizards/purchase_order_recommendation.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.tools.float_utils import float_compare, float_round
+from math import ceil
+
+
+class PurchaseOrderRecommendation(models.TransientModel):
+    _inherit = 'purchase.order.recommendation'
+
+    @api.model
+    def _prepare_wizard_line(self, vals, order_line=False):
+        res = super()._prepare_wizard_line(vals, order_line)
+        secondary_uom_id = (
+            order_line and order_line.secondary_uom_id or
+            vals.get('product_id') and
+            vals['product_id'].purchase_secondary_uom_id)
```

Comment:
This is a little bit hard to read. Maybe this would be better?

```python
secondary_uom_id = any(
    order_line and order_line.secondary_uom_id,
    vals.get('product_id') and vals['product_id'].purchase_secondary_uom_id,
)
```

Reply:
But I need the record, not a Boolean